### PR TITLE
d4-so called Life Disruptor

### PR DESCRIPTION
**Pull request by d4v3c**
tas amazingly amazing swep, by right click you can create a shield, the default ammo are ar2orbs, left click - shoot

### CODE REVIEW

**Comment by Derpius** [comment]
Can you add more details to the PR and remove non-ascii characters from the filename please?

**Review by d4v3c — COMMENTED** [review]
renamed

**Review by d4v3c — COMMENTED** [review]
removed the killicon

File: lua/weapons/d4-so called Life Disruptor [hunk 1]
```diff
@@ -0,0 +1,69 @@
+
+SWEP.PrintName			= "d4v3's so called Life Disruptor"
+SWEP.Author			    = "d4v3" 
+SWEP.Instructions		= "LMB TO SHOOT, RMB CREATES SHIELD"
+SWEP.ViewModel = "models/weapons/c_irifle.mdl"
+SWEP.WorldModel = "models/weapons/w_irifle.mdl"
+SWEP.ShootSound = Sound ( "Weapon_AR2.Single" )
```
Comment (Kilometres) [inline lua/weapons/d4-so called Life Disruptor]: nit: Spaced parentheses. (When you set up your workspace correctly, you won't need to worry about formatting.)

File: lua/weapons/d4-so called Life Disruptor [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+SWEP.Category = TASWeapons.Category
+SWEP.Primary.ClipSize = 25
+SWEP.Primary.DefaultClip = 60
+SWEP.Secondary.ClipSize = 5
+SWEP.Secondary.Ammo = ""
+SWEP.Secondary.DefaultClip = 5
+SWEP.Primary.Ammo = "AR2AltFire"
+SWEP.AdminSpawnable = true
```
Comment (Kilometres) [inline lua/weapons/d4-so called Life Disruptor]: "AdminSpawnable" is not a SWEP property used by Gmod, and you don't use it anywhere else. Could this be a mistake?

File: lua/weapons/d4-so called Life Disruptor [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        return
+    end
+    self:SetNextPrimaryFire(CurTime()+0.1)
+    self:ShootBullet(35,5,0.03)
+    self:EmitSound(self.ShootSound)
+    self:TakePrimaryAmmo(1)
+	end
+
+	function SWEP:SecondaryAttack()
+		if !CLIENT then
```
Comment (Kilometres) [inline lua/weapons/d4-so called Life Disruptor]: nit: Use `SERVER` not `!CLIENT`

File: lua/weapons/d4-so called Life Disruptor [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+			local ent = ents.Create("prop_physics")
+			ent:SetModel("models/hunter/tubes/tube2x2x2.mdl")
+			ent:SetPos(self.Owner:EyePos()+Vector(0, 0, -19))
```
Comment (Kilometres) [inline lua/weapons/d4-so called Life Disruptor]: `SWEP.owner` is deprecated, use `Entity:GetOwner()` instead.

**Comment by Derpius** [comment]
I'd generally dissuade leaving review comments on things that would be caught by the formatter/linter or CI in general, but in this case the pipeline is just completely broken for a PR made from a fork...

**Pull request closed without merging**